.github/workflows/deploy.yaml

```diff
@@ -57,4 +57,15 @@ jobs:
         run: docker push --all-tags $DOCKER_IMAGE
 
       - name: restart seqr instances
-        run: gcloud compute instance-groups managed rolling-action replace seqr-$DEPLOYMENT_TYPE-group --region australia-southeast1 --max-unavailable 0 --max-surge 3
+        run: |
+          gcloud compute instance-groups managed rolling-action replace \
+            seqr-$DEPLOYMENT_TYPE-group \
+            --region australia-southeast1 \
+            --max-unavailable 0 \
+            --max-surge 3
+
+      - name: restart seqr reference-updater
+          # used for cron jobs
+          gcloud run deploy \
```

Comment:
I assume this relies on having a different entrypoint defined in the original Cloud Run deployment?

Reply:
Yes, there's a new entrypoint already defined in the two cloud run instances:

```
python3 update_reference_server.py
```

It also has a subset of environment variables (+ secrets as env variables).